src/levels.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash';
+
+export let levels = {
```

Comment:
since these are default levels, please name the var accordingly (not the module)